### DISCUSSION TITLE
Version 11.1.3 Released

### DISCUSSION THREAD

**Discussion by maxkatz6** [AvaloniaUI/Avalonia#16674] [General]
### Features
* Added Compositor.CreateCompositionVisualSnapshot API by @kekekeks in https://github.com/AvaloniaUI/Avalonia/pull/16599
* [Android] Support `PerformContextMenuAction` by @Tangent-90 in https://github.com/AvaloniaUI/Avalonia/pull/15608
* Add api to allow text input method to trigger common context menu actions by @emmauss in https://github.com/AvaloniaUI/Avalonia/pull/15666

### Fixes
* [iOS] Switch to Metal by default by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/15994
* [Android] [TextBox] Fix a bug which cause cannot call out the soft keyboard again after closing it by @Tangent-90 in https://github.com/AvaloniaUI/Avalonia/pull/15603
* Suspend selection updates in TextBoxTextInputMethodClient until all changes are completed by @emmauss in https://github.com/AvaloniaUI/Avalonia/pull/15659
* [Windows] GetDpiForMonitor checks for Windows 8.0, but should check for 8.1 by @yusupovdanil in https://github.com/AvaloniaUI/Avalonia/pull/15734
* [Linux] Blacklist SVGA3D gpu driver by @kekekeks in https://github.com/AvaloniaUI/Avalonia/pull/15742
* Restore 0.10.x behavior for re-creating visual brush content on property change by @kekekeks in https://github.com/AvaloniaUI/Avalonia/pull/15838
* Prevent FontFamily Property was null and crash the whole application  by @Luoyingliang in https://github.com/AvaloniaUI/Avalonia/pull/15896
* Use the owner window's screen as the constraint. by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/15910
* Implemented VulkanDevice.Dispose by @jp2masa in https://github.com/AvaloniaUI/Avalonia/pull/15936
* [Windows] Fixed UIA pattern matching for IExpandCollapseProvider. by @boydpatterson in https://github.com/AvaloniaUI/Avalonia/pull/15997
* Fixed memory leak in Switch class by @ds1709 in https://github.com/AvaloniaUI/Avalonia/pull/15998
* [Windows] Restores WS_CHILD style for Embedded Windows by @fitdev in https://github.com/AvaloniaUI/Avalonia/pull/16051
* Fix compiled binding indexer always forcing integer boxing by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/16109
* [Windows] Patch Win32.TrayIconImpl DPI check for Win7/8 by @evgeniylevakhin in https://github.com/AvaloniaUI/Avalonia/pull/16148
* [Browser] Fix launcher on browser by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/16271
* [Windows] Disable WinFormsAvaloniaControlHost in design mode by @daniel-lerch in https://github.com/AvaloniaUI/Avalonia/pull/16296
* Fix - return (0,0) as size for text selector layer on MeasureOverride by @emmauss in https://github.com/AvaloniaUI/Avalonia/pull/16373
* [Linux] Compatible with "fcitx5" envrionment variable by @kkwpsv in https://github.com/AvaloniaUI/Avalonia/pull/16434
* [Windows] Win32.PlatformConstants uses buildNumber instead of version by @christophe-calmejane in https://github.com/AvaloniaUI/Avalonia/pull/16453
* [Android] Fix android uri activation by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/16477
* Fix TextLayout invalidation by @Gillibald in https://github.com/AvaloniaUI/Avalonia/pull/16481
* Remove internal access modifier from IWindowImpl.GetWindowsZOrder by @stevemonaco in https://github.com/AvaloniaUI/Avalonia/pull/16562
* Fixed overlay popups not automatically closing by @TomEdwardsEnscape in https://github.com/AvaloniaUI/Avalonia/pull/16564
* [Linux] Gracefully fall back to the next storage provider by @kekekeks in https://github.com/AvaloniaUI/Avalonia/pull/15929
* [Linux] Fix Linux X11 Atoms lost: Before the window was mapped, only the first atom was set. by @walterlv in https://github.com/AvaloniaUI/Avalonia/pull/16110

### Infrastructure
* Refactor integration test app paging by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16551
* Make integrational tests compatible with Appium 2 by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/15570
* Reset API validation baseline by @maxkatz6 in https://github.com/AvaloniaUI/Avalonia/pull/16470
* Record video for failing win32 integration tests on CI. by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16546
* Try to fix flaky window decorations tests on Windows. by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16597

**Full Changelog**: https://github.com/AvaloniaUI/Avalonia/compare/11.1.2...11.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.1.3'>11.1.3</a>.</em>

**Comment by Dajonas14** [comment]
Cool
